src/Movies.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import MovieCard from './MovieCard';
 import './Movies.css';
 
-function Movies() {
+function Movies({moviesInfo}) {
+    const film = moviesInfo.map(movie => {
+      return (
+        <MovieCard
+            title = {movie.title}
+            posterPath = {movie.poster_path}
+            averageRating = {movie.average_rating}
```

Comment:
this is where we would add logic for determining the border color I think?

Reply:
How are you determining border color? Remember to avoid inline styling if you can! All styling should be in your CSS file. You can add dynamic classes tho!